BossMod/Components/CastHint.cs

```diff
@@ -3,6 +3,19 @@
 
 namespace BossMod.Components
 {
+    public enum AID : uint
```

Comment:
This will pollute very common namespace (Components) with a very common type name (AID) that only actually contains very specific action id's relevant only to one component (and even then - only for AI of all things). This should either be a private subtype of interrupt hint component, or moved to proper location (eg. Autorotation/BLU/...).

Also, this should probably be moved to a different PR, as it's not really related to zadnor CE's, as PR name implies.

Reply:
I will see what I can do. Sadly, noone made anything for BLU autorotation yet, so I had this idea since I have no plans to do any work on autorotations.